src/pages/our-team.js

```diff
@@ -1,14 +1,35 @@
 import React from "react"
-import { Link } from "gatsby"
 import { graphql } from "gatsby"
 import get from "lodash/get"
 import Layout from "../components/layout/layout"
 import SEO from "../components/seo"
+import Team from "../components/teampage/team-members/teams"
+import Advisor from "../components/teampage/advisors/advisors"
 import Header from "../components/header/header"
 
 const TeamPage = props => {
   const teamPage = get(props, "data.contentfulTeamPage")
 
+  const listOfTeamsTypes = getTeamTypes(teamPage.teamMembers)
+
+  /**
+   * creates a list/array of all the team types
+   * @param {Object[]} members - Array of members
+   * @param {string} members[].type - the team the member is on
+   * @returns {string[]} - Array/list of string member types ie ['board', 'marketing']
+   */
+  function getTeamTypes(members) {
```

Comment:
This function could be reduced to `members.filter(m => !m.team).map(m => m.team.toLowercase())`